apps/examples/timer/timer_main.c

```diff
@@ -253,8 +236,12 @@ int timer_main(int argc, char *argv[])
 		repeat = EXAMPLE_TIMER_NSAMPLES;
 	}
 
-	/* Create Time Handler Thread */
-	tid = create_timer_thread((void *)repeat);
+	sigemptyset(&sigset);
+	sigaddset(&sigset, EXAMPLE_TIMER_SIGNO);
+	sigprocmask(SIG_BLOCK, &sigset, NULL);
```

Comment:
Why is this necessary here?

Reply:
You are right. This is not necessary, we are explicitly creating the signal set in the timer thread. Cleanup done.